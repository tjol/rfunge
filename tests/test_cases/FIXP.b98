"PXIF"4(v                           >5a*aa**1--;@,kd"unknown error"a;# _a"gnidnuor dab"ck,@
v       > f2* a:*:** I : 5a*aa**-  #^_a"ko nis"6k, ; sin 30° = 0.5 ;
> J ; arcsin ; C: ; cos 30° * 1e4 = sqrt (3e8) / 2 ; v
 @_v#-*a+2*9`' ; which is 8660 @_ #;<; - /2Q*3*:*:*:a<<<<
 v >5a*aa**B ; arccos 0.5 = 60 ; 6'd*a'd**-#@_ a"ko soc & trqs"dk,
v>'d:*:U: '-'d:** - #@_ ; atan(1) = 45° ; T-#@_ a"ko nat"6k,
> 07-7N-#@_ bNVb-#@_ 3S1-#@_04-S1N-#@_ a"ko .oc & ngs"ck, v
          @ _ #;  a"ko esiwtib"ak,  v; -*73 X3  O*28 Ae7  <
                                    > 2aa**P 'N8*4+ - #@_a"ko ip"5k,@