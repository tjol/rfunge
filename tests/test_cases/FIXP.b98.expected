sin ok
sqrt & cos ok
tan ok
sgn & co. ok
bitwise ok
pi ok
